package.json

```diff
@@ -28,10 +28,12 @@
     "all": true
   },
   "scripts": {
+    "lint": "eslint .",
     "lint:types": "dtslint types && eslint types/index.d.ts",
     "lint:fix": "eslint --fix .",
     "test": "mocha test/**/*-test.js && npm run lint:types",
     "test:coverage": "nyc mocha test/**/*-test.js && npm run lint:types",
+    "test:ci": "npm run lint && npm run lint:types && npm run test:coverage",
```

Comment:
imo, it's better to separate between test and lint jobs in travis.

what do you say?

Reply:
sure, why not